### PR TITLE
Adds two different routes for sign-up:

### PR DESCRIPTION
**Pull request by heridev**
Routes added for creating different type of users:   /nurses/sign_up and - /customers/sign_up

- Customizes login, signup and forgot password devise's views
- Allows type field param when the users signs up
- Adds login and logout options in the home page
- Changes title name and icon in registration page depending on type of user
- Creates different type of users using STI through the usage of the field "type"
- Sets customer as a default user type when is not provided.
- Adds spanish translations

### CODE REVIEW

**Comment by jtapia** [comment]
LGTM

File: app/views/layouts/application.slim [hunk 1]
```diff
@@ -11,10 +11,14 @@ html
...
           li
-            a href="#"  Iniciar Sesi&oacute;n
+            - if user_signed_in?
+              = link_to('Cerrar Sesión', destroy_user_session_path, :method => :delete)
+            - else
+              = link_to "Iniciar sesión", new_user_session_path
+
         ul#nav-mobile.side-nav
           li
             a href="#"  Iniciar Sesi&oacute;n
```
Comment (gus4no) [inline app/views/layouts/application.slim]: you should use the same logic to log in and out in this line which belongs to mobile layout

Comment (heridev) [inline app/views/layouts/application.slim]: that's right I forgot that section, thanks

**Comment by gus4no** [comment]
LGTM

**Pull request merged**